src/xtgeo/grid3d/_grid_etc1.py

```diff
@@ -576,6 +576,63 @@ def get_xyz_corners(self, names=("X_UTME", "Y_UTMN", "Z_TVDSS")):
     return tuple(grid_props)
 
 
+def get_vtk_esg_geometry_data(
+    self,
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    """Get geometry data consisting of vertices and cell connectivities suitable for
+    use with VTK's vtkExplicitStructuredGrid.
+
+    Returned tuple contains:
+     - dims in terms of points
+     - vertex array
+     - connectivities
+     -  inactive cell indieces
+    """
```

Comment:
Be a bit more explicit on explaining returns and correct typos

Reply:
Fixed typo and added some more description to the returned tuple